samples/ClientCredentialsFlow/ClientApp/ClientApp.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
We should revert that. It's also worth noting that OpenIddict 3.0 won't use JSON.NET, so we'll have to remove it at some point.